main/dovecot/APKBUILD

```diff
@@ -3,11 +3,11 @@
 # Contributor: Michael Mason <ms13sp@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=dovecot
-pkgver=2.2.27
+pkgver=2.2.29.1
 pkgrel=2
-_pigeonholever=0.4.16
+_pigeonholever=0.4.18
 _pluginextdataver=39
-_pkgvermajor=${pkgver%.*}
+_pkgvermajor=$(echo $pkgver | cut -d. -f1-2)
```

Comment:
I'll see if I can get a param expansion version working. (The previous version didn't cope with a changing number of dots in the version, i.e. it went from 2.2.28 to 2.2.29.1)